src/vs/editor/common/services/languageService.ts

```diff
@@ -94,20 +98,20 @@ export class LanguageService extends Disposable implements ILanguageService {
 		return firstOrDefault(languageIds, null);
 	}
 
-	public createById(languageId: string | null | undefined): ILanguageSelection {
+	public createById(languageId: string | null | undefined): ILanguageSelection {//
```

Comment:
What is the meaning of `//` at the end of the line?

Reply:
An oversight, I just left a marker to find all references to double check something